test/clojure-mode-indentation-test.el

```diff
@@ -68,6 +68,7 @@ values of customisable variables."
            (search-forward "|")
            (delete-char -1)
            (clojure-mode)
+           (font-lock-ensure)
```

Comment:
How can this affect an indentation test?

Reply:
I think it is because the indentation command behaves differently depending on whether font lock mode is ON or OFF, in the case of docstrings.

This branch is master with the test added: https://github.com/carlosgeos/clojure-mode/tree/temp-branch. The tests are fine, but if `(font-lock-ensure)` is added, the test with the docstring fails (like it should).

Reply:
Hmm, that’s odd. Turning on the mode should enable the font locking.

Reply:
Usually yes, but apparently not when using temp-buffer:

https://stackoverflow.com/a/23569337
https://emacs.stackexchange.com/a/44301